tests/python/unittest/test_auto_scheduler_compute_dag.py

```diff
@@ -23,7 +23,7 @@
 from tvm import topi
 from tvm import auto_scheduler, te
 
-from test_auto_scheduler_common import (
+from tvm.testing.auto_scheduler_common import (
```

Comment:
let's name it `tvm.testing.auto_scheduler`

Reply:
done